atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -827,15 +773,30 @@ private TableMetadata getTableMetadataForSweepStrategy(SweepStrategy sweepStrate
                 false);
     }
 
-    private void testValidateExternalAndCommitLocks(final LockAwareTransactionTask<Void, Exception>
-            lockAwareTransactionTask) throws Exception {
+    private void runTaskWithInvalidLocks(final LockAwareTransactionTask<Void, Exception> lockAwareTransactionTask)
+            throws Exception {
         keyValueService.createTable(TABLE_SWEPT_THOROUGH, getTableMetadataForSweepStrategy(
                 SweepStrategy.THOROUGH).persistToBytes());
         txManager.runTaskWithLocksThrowOnConflict(
-                ImmutableList.of(getFakeHeldLocksToken()),
+                ImmutableList.of(getExpiredHeldLocksToken()),
                 lockAwareTransactionTask);
     }
 
+    private HeldLocksToken getExpiredHeldLocksToken() {
+        ImmutableSortedMap.Builder<LockDescriptor, LockMode> builder =
+                ImmutableSortedMap.naturalOrder();
+        builder.put(AtlasRowLockDescriptor.of(TransactionConstants.TRANSACTION_TABLE.getQualifiedName(),
+                TransactionConstants.getValueForTimestamp(0L)), LockMode.WRITE);
```

Comment:
I didn't notice this before, but when you have a long list of parameters split across multiple lines, we prefer to have one parameter per line (also below).